src/components/PlanCard/SharePlanModal.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { Modal } from '../../components/Modal/Modal';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 1em;
+`;
+
+const Text = styled.h4`
+  display: flex;
+  gap: 1em;
+  align-items: center;
```

Comment:
i dont think this needs to be a flex, you can just add the `align-items: center` in `Container` to center align the text